R/geom-text-repel.R

```diff
@@ -446,6 +448,11 @@ makeContent.textrepeltree <- function(x) {
         segment.curvature = row$segment.curvature,
         segment.angle     = row$segment.angle,
         segment.ncp       = row$segment.ncp,
+        segment.shape = row$segment.shape,
```

Comment:
I was not sure if I should align with whitespaces here - the two lines above are the only two doing so.